lambdas/backend-api/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.1",
   "scripts": {
     "generate-dependencies": "tsx scripts/generate-dependencies.ts",
-    "test:unit": "jest",
+    "test:unit": "npm run generate-dependencies && jest",
```

Comment:
if we're running generate dependencies as part of the test:unit commands, we don't need to run it separately on the pipeline right? no strong views which one we keep but feels like we could save time by not having both

Reply:
I added it here because I ran the test script locally and it failed because I hadn't run the script. I think it was the first time I'd run it since the workspaces thing came in and the file structure was changed. I added it because you need the dependency to make sure the tests pass, so it made sense to make the test command run it. 

But I've just realised that the linting and typecheck also fail if you don't have the `email-template.json` file. So with this approach, really I should be prefixing all of those commands with `generate-dependencies`. Which does seem a bit pointless - it'd be better to stick with the current approach and run it once manually when needed. 

And also there are a few packages that have the `generate-dependencies` script, and the pipelines are running it in all the workspaces. Whereas this is just in the `backend-api` package. So we'd still need to keep it in the pipeline for those other packages possibly.

So in conclusion, I will remove this!

Reply:
Updated